decidim-core/db/migrate/20161020080756_add_position_to_steps.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToSteps < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_process_steps, :position, :integer
```

Comment:
We can add validations allowing blank values. Regarding default values, I'd expect new steps to be added to the end, not the begining. Maybe the default value can be the previous + 1 (`resort` solves this 😝)